### PR TITLE
Adjusting changes for crab namespace.

### PR DESCRIPTION
**Pull request by arooshap**

### CODE REVIEW

File: helm/cmsweb/crabcache/templates/configMap.yaml [hunk 1]
```diff
@@ -34,7 +34,9 @@ metadata:
 data:
   setup-certs-and-run.sh: |
     #!/bin/bash
-    cp /host/etc/grid-security/* /etc/grid-security
+    sudo cp /host/etc/grid-security/* /etc/grid-security
     echo 'INFO Files in /etc/grid-security'
     ls -lahZ /etc/grid-security
     su  -c "cd /data && /data/run.sh" --preserve-environment  _crabcache
```
Comment (muhammadimranfarooqi) [inline helm/cmsweb/crabcache/templates/configMap.yaml:40]: @arooshap 
This line  `su  -c "cd /data && /data/run.sh" --preserve-environment  _crabcache` should be commented.

File: helm/cmsweb/crabcache/values.yaml [hunk 2]
```diff
@@ -106,25 +113,7 @@ serviceMon:
   protocol: TCP
   name: crabcache-mon
  
-command:
-- /bin/bash
-- /opt/setup-certs-and-run/setup-certs-and-run.sh
```
Comment (muhammadimranfarooqi) [inline helm/cmsweb/crabcache/values.yaml:111]: @arooshap 

This command (` - /opt/setup-certs-and-run/setup-certs-and-run.sh`) should not be deleted. This is in main container.

Comment (arooshap) [reply helm/cmsweb/crabcache/values.yaml:111]: It wasnt deleted, it was shifted upwards.
image

File: helm/cmsweb/crabcache/templates/configMap.yaml (hunk 1 above)
Comment (arooshap) [reply helm/cmsweb/crabcache/templates/configMap.yaml:40]: https://github.com/arooshap/CMSKubernetes/blob/a70bf64fb9597f12e8e5cddb0b166c0a7d2f2f9d/kubernetes/cmsweb/services/crabcache.yaml#L57

This needs to be updated as well.

Comment (muhammadimranfarooqi) [reply helm/cmsweb/crabcache/templates/configMap.yaml]: yes, I have fixed there as well. Thanks

**Pull request merged**